source/jormungandr/jormungandr/parking_space_availability/bss/bss_provider_manager.py

```diff
@@ -111,11 +112,13 @@ def _handle_poi(self, item):
                 return provider
         return None
 
-    # TODO use public version everywhere
     def _get_providers(self):
         self.update_config()
-        # providers from the database have priority on legacies providers
         return list(self._bss_providers.values()) + self._bss_providers_legacy
 
     def get_providers(self):
         return self._get_providers()
+
+    def exist_provider(self):
+        self.update_config()
+        return any((self._bss_providers.values(), self._bss_providers_legacy))
```

Comment:
If think it will return true if both are empty lists, no?
```suggestion
        return any(self.get_providers())
```

Reply:
:+1: